source/common/config/version_converter.cc

```diff
@@ -1,43 +1,75 @@
 #include "common/config/version_converter.h"
 
+#include "common/common/assert.h"
 #include "common/config/api_type_oracle.h"
 
 namespace Envoy {
 namespace Config {
 
 namespace {
 
+// TODO(htuch): make the unknown field validators aware of this distinguished
+// field, and don't reject if present.
+constexpr uint32_t OriginalTypeFieldNumber = 100000;
```

Comment:
Yeah, unfortunately I can't do much with this until I can build and run without

C++ exception with description "Protobuf message (type envoy.config.route.v3alpha.RouteConfiguration with unknown field set {100000}) has unknown fields" thrown in the test body.

all over :-(